src/Compilers/CSharp/Test/Emit2/CodeGen/CodeGenMethodGroupConversionCachingTests.cs

```diff
@@ -126,6 +126,323 @@ .maxstack  4
 ");
     }
 
+    [Fact]
+    public void Not_DelegateCreations_CapturingLocalFunction()
+    {
+        var source = @"
+using System;
```

Comment:
Maybe use raw strings for new test cases? Esp. considering that IL has a lot of quotes, which can be simplified then

Reply:
No problem, I can create a follow-up pr for the entire file.

Reply:
Compiler team doesn't like such "style-only" changes. I remember when I did such thing to syntax normalizer tests, it wasn't taken with ease. So it's probably a good idea to wait for someone from the team to tell, what is better to be done here

Reply:
Yes, merge conflicts could be a huge headache. Let's keep it consistent for now. But this file was created for delegate caching and only a few edits were made since then.

Reply:
> Let's keep it consistent for now

You will probably never experience new language features in huge codebases like rolsyn with this mindset) No preasure though, such decisions are up to you and code owners and I am just a random person, so feel free to ignore my opinion here)